src/index.js

```diff
@@ -2,14 +2,16 @@ function toVal(mix) {
 	var k, y, str='';
 
 	if (typeof mix === 'string' || typeof mix === 'number') {
-		str += mix;
-	} else if (typeof mix === 'object') {
+		return mix;
+	}
+	
+	if (typeof mix === 'object') {
 		if (Array.isArray(mix)) {
-			for (k=0; k < mix.length; k++) {
-				if (mix[k]) {
-					if (y = toVal(mix[k])) {
+			for (y=0; y < mix.length; y++) {
```

Comment:
Why did these vars get swapped?

Reply:
Codebase contains
```
str && (str += ' ');
str += k;
```
thrice. I try to create more identical pieces of code for better work for the "deflate" algorithm. This trick saves 2 bytes of gzip size. I don't rename the third piece(https://github.com/lukeed/clsx/blob/master/src/index.js#L34) because the total size not changed.